ddtrace/contrib/grpc/utils.py

```diff
@@ -0,0 +1,7 @@
+def parse_method_path(method_path):
+    # unpack method path based on "/{package}.{service}/{method}"
```

Comment:
I _think_ some of these are optional, no?

I thought `package` and `service` were not required?